common/src/main/java/org/opensearch/ml/common/model/Guardrails.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import java.io.IOException;
+
+import static org.opensearch.core.xcontent.XContentParserUtils.ensureExpectedToken;
+
+@EqualsAndHashCode
+@Getter
+public class Guardrails implements ToXContentObject {
+    public static final String TYPE_FIELD = "type";
+    public static final String ENGLISH_DETECTION_ENABLED_FIELD = "english_detection_enabled";
+    public static final String INPUT_GUARDRAIL_FIELD = "input_guardrail";
+    public static final String OUTPUT_GUARDRAIL_FIELD = "output_guardrail";
+
+    private String type;
```

Comment:
What kind of values are we expecting for the Guardrail type?

Reply:
Reserve this type field to handle future different guardrail schema. Currently we only have local_regex type.

Reply:
are we expecting this input from customer now?

Reply:
No, cx don't have to configure it right now.